src/windows/views/blender_listview.py

```diff
@@ -708,10 +703,14 @@ class Worker(QObject):
     blender_error_with_data = pyqtSignal(str)  # 1007
     enable_interface = pyqtSignal()  # 1008
 
+    @pyqtSlot()
+    def Cancel(self):
+        """Cancel worker render"""
+        self.is_running = False
+
```

Comment:
Let's try that again:

Would this let blender run until it finishes its current job? Could it actually do a self.process.terminate() as well, to pull the worker out early?

Reply:
Yes, good idea. Done.